lib/engine/game/base.rb

```diff
@@ -42,6 +42,13 @@ class Base
 
       HEXES = {}.freeze
 
+      CUSTOM_TILES = {
```

Comment:
i feel like we could treat this like we do hexes inside tiles and have inline definition for custom tiles, i think that'd be closer to @kelsin 18xxmaker

Reply:
Updated in d8f4da2 and later commits